datasets/batch_view/client_count/intro.md

```diff
@@ -1,14 +1,18 @@
 The `client_count` dataset is useful for estimating user counts over a few
 [pre-defined dimensions](https://github.com/mozilla/telemetry-airflow/blob/master/jobs/client_count_view.sh).
 
+The `client_count` dataset is deprecated in favor of
+[client_count_daily](../client_count_daily/reference.md),
```

Comment:
Please wrap client_count_daily with backticks instead of adding a dictionary entry. E.g. `client_count_daily`

Reply:
✅  Thanks!